data.sql

```diff
@@ -8,4 +8,28 @@ INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
 VALUES ('Pikachu', '2021/01/07', 1, false, 15.0);
 
 INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
-VALUES ('Devimon', '2017/05/12', 5, true, 11.0);
\ No newline at end of file
+VALUES ('Devimon', '2017/05/12', 5, true, 11.0);
+
+
+--Insert new data 
+
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg, species)
+VALUES ('Charmander', '2020-02-08', 0, FALSE, -11, 'Fire Type');
+
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg, species)
+VALUES ('Plantmon', '2021-11-15', 2, TRUE, -5.7, 'Grass Type');
+
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg, species)
+VALUES ('Squirtle', '1993-04-02', 3, FALSE, -12.13, 'Water Type');
+
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg, species)
+VALUES ('Angemon', '2005-06-12', 1, TRUE, -45, 'Holy Type');
+
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg, species)
+VALUES ('Boarmon', '2005-06-07', 7, TRUE, 20.4, 'Beast Type');
+
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg, species)
+VALUES ('Blossom', '1998-10-13', 3, TRUE, 17, 'Flower Type');
+
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg, species)
+VALUES ('Ditto', '2022-05-14', 4, TRUE, 22, 'Transforming Type');
```

Comment:
Hi, thank you for your submission. I have one required comment and some optional suggestions on your code:

- **Required**: You added a species column to the INSERT INTO statements, which was not part of the instructions. The instructions only asked for name, date_of_birth, escape_attempts, neutered, and weight_kg columns. Please remove the species column from your code and resubmit.


Please make the required change and resubmit your code for another review. If you have any questions or need any help, feel free to reach out to me. Thank you for your hard work and cooperation.